### DISCUSSION TITLE
Unknown section type

### DISCUSSION THREAD

**Discussion by bert-laverman** [albar965/littlenavmap#674] [Q&A]
Using the documentation I found at several locations, including [this link at FSDeveloper](https://www.fsdeveloper.com/wiki/index.php/BGL_File_Format), I am progressing well with a basic BGL reader. I did immediately turn up a new section type 170 (0xAA). Does anyone know what this type stands for?

**Comment by albar965** [comment]
Hi Bert,

sorry, this one is unknown to me. 
Section types used by LNM are here: https://github.com/albar965/atools/blob/release/3.6/src/fs/bgl/sectiontype.h

Alex

**Comment by albar965** [comment]
Something more: You normally would like to skip all unknown sections and records since P3D or MSFS usually add new ones if structures are modified.
